src/mmw/apps/core/decorators.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from django.utils.timezone import now
+
+from apps.core.models import (Job,
+                              RequestLog)
+
+
+def log_request(view):
+    """
+    Log the request and its response as a RequestLog model
+    """
+
+    def decorator(request, *args, **kwargs):
+        requested_at = now()
+
+        view_result = view(request, *args, **kwargs)
+
+        user = request.user if request.user.is_authenticated() else None
+
+        job_uuid = view_result.data.get('job', None)
+        job = None
+        if (job_uuid):
+            try:
+                job = Job.objects.get(uuid=job_uuid)
+            except Job.DoesNotExist:
+                pass
+
+        response_time = now() - requested_at
+        response_ms = int(response_time.total_seconds() * 1000)
+
+        log = RequestLog.objects.create(
+            user=user,
+            job=job,
+            requested_at=requested_at,
+            response_ms=response_ms,
+            status_code=view_result.status_code,
+            path=request.path,
+            query_params=request.query_params.dict(),
+            method=request.method,
+            host=request.get_host(),
+            remote_addr=_get_remote_addr(request))
+        try:
+            log.save()
+        except Exception:
+            pass
```

Comment:
Rather than silently failing, is it possible for this to alert Rollbar of a failure, but still succeeding the request?